README.md

```diff
@@ -89,12 +89,31 @@ installer.on('state-changed', ({version, state}) => {
 await installer.ensureDownloaded('12.0.15');
 // expect(installer.state('12.0.5').toBe('downloaded');
 
+// download a version with callback
+const callback = (progress: ProgressObject) => {
+  const percent = progress.percent * 100;
+  console.log(`Current download progress %: ${percent.toFixed(2)}`);
+};
+await installer.ensureDownloaded('12.0.15', {
+  progressCallback: callback,
+});
+
+// download a version with a specific mirror
+const npmMirrors = {
+  electronMirror: 'https://npmmirror.com/mirrors/electron/',
+  electronNightlyMirror: 'https://npmmirror.com/mirrors/electron-nightly/',
+},
+
+await installer.ensureDownloaded('12.0.15', {
+  mirror: npmMirrors,
+});
+
 // remove a download
 await installer.remove('12.0.15');
 // expect(installer.state('12.0.15').toBe('not-downloaded');
 
 // install a specific version for the runner to use
-const exec = await installer.install('11.4.10');
+const exec = await installer.install('11.4.10'); // Additional params can be passed here as well like shown in `ensureDownloaded` function
```

Comment:
Instead of a comment here, could we turn this into a new paragraph with an example of using `ensureDownloaded`? I think erring on the side of more verbose documentation/examples is better when we can

Reply:
Cool done that to make it more clear! Btw, you mean `installer.install` example right ? (Since, I have already added the example of `ensureDownloaded` above)